.jenkins/build_docker_images.sh

```diff
@@ -1,23 +1,25 @@
 #!/bin/bash
 
+# Read in tag name used for the Docker images to be tested
+tagname=$1
+
 # Set variable to allow parallel building in the Docker image creation
 export DOCKER_BUILDKIT=1
 export BUILDKIT_PROGRESS=plain
-targets="build build_serialize_gt4py_dev"
-tags="hpc hpc-serialize"
 
 # Speed-up the compilations by using pre-built MPI, FMS, and ESMF images
 export BUILD_FROM_INTERMEDIATE=y
 
 # Build FV3 without and with Serialbox support enabled
 make pull_deps
-COMPILED_TAG_NAME=hpc make ${targets} 
+make DEP_TAG_NAME=$tagname build build_serialize 
 
 # For each newly built Docker image:
 #   - push image to VCM's Google Container Repository (necessary?)
 #   - create a tar archive of the image
 #   - store tar archive in a Google Storage Bucket
-for tag in ${tags} ; do
+declare -a tags=("$tagname" "${tagname}-serialize")
+for tag in ${tags}; do
```

Comment:
Could be safer to use `... in ${tags} ; do`? See issue of Jeremy doing `if [ ${tags}] ; then` and getting an error. Space matters in bash...